plat/kvm/arm/entry64.S

```diff
@@ -49,7 +49,7 @@ lcpu_bootstack:
  */
 
 .text
-ENTRY(_libkvmplat_entry)
+ENTRY(qemu_virt_entry)
```

Comment:
In anticipation of Firecracker support on arm64, please refrain from renaming this to QEMU virt

Reply:
Could you give me a suggestion? Maybe just `virt`? :thinking:

Reply:
Just revert. Because now that we will be supporting both QEMU virt and Firecracker, KVM is technically correct. On re-arch we will break this into individual platforms.